src/main/resources/application-dev.properties

```
@@ -0,0 +1,16 @@
+spring.datasource.url=jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}?user=${DB_USERNAME}&password=${DB_PASSWORD}
```

pra mim só funcionou com o `sslmode`
```suggestion
spring.datasource.url=jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}?user=${DB_USERNAME}&password=${DB_PASSWORD}&sslmode=require
```

---

Que estranho... Pra mim só funcionou sem. Vou colocar de volta então.